src/core/updateWebpackConfig.js

```diff
@@ -13,31 +13,25 @@ const updateWebpackConfig = (webpackConfig, env, serverInfo) => {
     {
       test: /\.jsx?$/,
       include: srcFolder,
-      loader: 'babel',
+      loader: 'babel-loader',
       query: {
+        babelrc: false,
         presets: [
           'es2015',
           'react',
         ],
```

Comment:
I think this is where the problem is.  I don't think the babel loader should be specified in snapguidist, we should use the user's .babelrc file.